app/src/main/kotlin/amhk/chronos/ui/BlockListFragment.kt

```diff
@@ -38,7 +40,7 @@ internal class BlockListFragment : Fragment() {
                               savedInstanceState: Bundle?): View {
         val view = inflater.inflate(R.layout.fragment_block_list, container, false)
 
-        view.findViewById<RecyclerView>(R.id.block_list).also {
+        block_list.also {
```

Comment:
This causes a null pointer exception. I will remove this patch from the patch chain and merge the rest, and you can work on re-rolling this patch in a new pull request.

This is the stack trace:
```
FATAL EXCEPTION: main
Process: amhk.chronos, PID: 19006
java.lang.RuntimeException: Unable to start activity ComponentInfo{amhk.chronos/amhk.chronos.ui.MainActivity}: java.lang.IllegalStateException: it must not be null
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2680)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2741)
        at android.app.ActivityThread.-wrap12(ActivityThread.java)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1488)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:154)      
        at android.app.ActivityThread.main(ActivityThread.java:6166)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:889)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779)
Caused by: java.lang.IllegalStateException: it must not be null
        at amhk.chronos.ui.BlockListFragment.onCreateView(BlockListFragment.kt:44)
        at android.support.v4.app.Fragment.performCreateView(Fragment.java:2343)
        at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1421)
        at android.support.v4.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManager.java:1752)
        at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1821)
        at android.support.v4.app.BackStackRecord.executeOps(BackStackRecord.java:797)
        at android.support.v4.app.FragmentManagerImpl.executeOps(FragmentManager.java:2595)
        at android.support.v4.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2382)
        at android.support.v4.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManager.java:2337)
        at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:2244)
        at android.support.v4.app.FragmentManagerImpl.dispatchStateChange(FragmentManager.java:3255)
        at android.support.v4.app.FragmentManagerImpl.dispatchActivityCreated(FragmentManager.java:3205)
        at android.support.v4.app.FragmentController.dispatchActivityCreated(FragmentController.java:195)
        at android.support.v4.app.FragmentActivity.onStart(FragmentActivity.java:597)
        at android.support.v7.app.AppCompatActivity.onStart(AppCompatActivity.java:177)
        at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1248)
        at android.app.Activity.performStart(Activity.java:6699)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2643)
        ... 9 more
```